css/css-flexbox/percentage-padding-002.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<title>This test checks that percentage paddings do not lead to flex items with negative sizes.</title>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-3/#padding-physical" title="Percentages line">
+<link rel="match" href="scrollbars-ref.html">
+<style>
+div {
+    width: 50px;
+    height: 50px;
+}
+span {
+    display: inline-flex;
```

Comment:
Does this actually require both spans to be inline-flex and have the % padding?

Reply:
Yeah both should be `inline-flex`. Regarding the % padding we can leave it just in the inner span.